src/config.js

```diff
@@ -2,12 +2,25 @@ import Dotenv from 'dotenv';
 
 Dotenv.config();
 
-export const {
-  NAME = 'Graasp Printer',
+
+const {
   PORT = 9696,
   GRAASP_HOST = 'https://graasp.eu',
   LOGGING_LEVEL = 'info',
   TMP_PATH = 'tmp',
+  DOCUMENTS_HOST = 'http://localhost:3000',
+  S3_BUCKET = 'graasp-files',
+  S3_HOST = 'https://graasp-files.s3.amazonaws.com',
```

Comment:
Default `S3_HOST` to `null`.

Reply:
We want to only include the defaults that would work for any user. The bucket and s3 host are protected, so we should not include those here. Default them to `null` and pass them through a `.env.local` file to test.